src/editor/components/modals/GeoModal/GeoModal.component.jsx

```diff
@@ -0,0 +1,151 @@
+import { useState, useCallback, useEffect } from 'react';
+
+import styles from './GeoModal.module.scss';
+import { Copy32Icon, Mangnifier20Icon } from '../../../icons';
+
+import { useAuthContext } from '../../../contexts/index.js';
+import Modal from '../Modal.jsx';
+import { Button, Input } from '../../components/index.js';
+import { GoogleMap, useJsApiLoader, Marker } from '@react-google-maps/api';
```

Comment:
Including this new @react-google-maps/api dependency increased the bundle size to be above 1.4MB. We should start using some code splitting in this project so the user don't have to download all the js up front. And also this feature is actually for pro user, so free user shouldn't need to download that code at all.
You can lazy load the GeoModal component, see https://react.dev/reference/react/lazy

Reply:
Thanks. Then maybe does it make sense to do this for the aframe-mapbox-component and aframe-loader-3dtiles-component libraries too?

Reply:
Absolutely. We can do that later i needed. If you want to merge quickly the changes, you can increase maxAssetSize and maxEntrypointSize to 1572864 (1.5MB) in webpack.prod.config.js

Reply:
We won't do this in this PR, see https://github.com/3DStreet/3dstreet/issues/524#issuecomment-2141897553